test-cases/jetpack/videopress.md

```diff
@@ -28,21 +28,6 @@ Use Simple WordPress.com site with a Premium plan. Free or Personal plans only a
 
 <img src="../../test-cases/resources/../resources/videopress-upload-video-4.png" width="360">
 
-##### TC002
```

Comment:
This test case was automated in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5826.